src/lib/ledgers/multiledger.js

```diff
@@ -1,38 +1,51 @@
 'use strict'
 
 const FiveBellsLedger = require('./five-bells-ledger')
+const healthStatus = require('../../common/health.js')
 
 function Multiledger (options) {
   this.config = options.config
   this.log = options.log
   this.ledger_types = {undefined: FiveBellsLedger}
   this.ledgers = {}
+  this.buildLedgers()
```

Comment:
The previous build ledger didn't mutate the instance, it just returned a new ledger. So maybe, `this.ledgers = buildLedgers()` is closer to that.